src/Illuminate/Cache/DatabaseStore.php

```diff
@@ -169,6 +170,7 @@ public function put($key, $value, $seconds)
     /**
      * Store multiple items in the cache for a given number of seconds.
      *
+     * @param  array  $values
      * @param  int  $seconds
```

Comment:
This probably as well:
```suggestion
     * @param  non-negative-int  $seconds
```